src/transaction/log_page_async_fetcher.cpp

```diff
@@ -38,10 +38,6 @@ namespace cublog
       async_page_fetcher::callback_func_type m_callback;
   };
 
-  /*********************************************************************
```

Comment:
Any reason that this comment was removed?

Reply:
I added it in this PR, then I removed it as part of one of the PR's disjoined from this one.